webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
```

Comment:
Good job working on the project requirements. Its against microverse policy to disable eslint or any other linters because linters help us write clean code. it would be nice if you enable eslint.